general/documentation-day.md

```diff
@@ -0,0 +1,29 @@
+# Documentation Day
+
+## Goals of the Documentation Day
+
+The general idea is that every project has a regular day when the documentation is checked. The goal is to have **well maintained documentations with all necessary and up-to-date information**.
+
+The main goal of the Documentation Day is to **establish and maintain confidence in the documentation**.
+
+## Where to Start
+
+Assigning all involved developers to execute a Documentation Day, one after another, is a good idea. Introducing new developers by having them execute the Documentation Day will provide great insight and valuable feedback, as well as asking developers who are not involved in the project.
+
+The following steps are one possible way for the Documentation Day:
+
+tbd
+
+If possible, fix all identified problems directly and update the documentation regardingly. Otherwise try to schedule the issues to have them fixed at least until the next Documentation Day.
```

Comment:
Maybe it's good to state some examples of possible problems to have an orientation.

Reply:
Oops, forgot to add an example here. They're added now.